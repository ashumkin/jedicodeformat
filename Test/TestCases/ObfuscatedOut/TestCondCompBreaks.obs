unit testcondcompbreaks;interface implementation uses sysutils,{$IFDEF BAR_RAISED}
 Classes, {$ENDIF}dialogs;{$IFDEF SYM2} const
  SOCK_IT_TO_ME = 'Black socks'; {$ENDIF}
procedure somestuff;begin showmessage('Some stuff...');end;procedure morestuff;begin
{$IFDEF HAS_STUFF}
  SomeStuff;
  {$ENDIF}end;end.