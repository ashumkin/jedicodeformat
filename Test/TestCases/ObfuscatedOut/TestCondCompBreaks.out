unit testcondcompbreaks;

interface

implementation

uses SysUtils,{$IFDEF BAR_RAISED}
 Classes, {$ENDIF}Dialogs;

{$IFDEF SYM2} const
  SOCK_IT_TO_ME = 'Black socks'; {$ENDIF}

procedure somestuff;
begin
  ShowMessage('Some stuff...');
end;

procedure morestuff;
begin
{$IFDEF HAS_STUFF}
  SomeStuff;
  {$ENDIF}
end;

end.
